frontend/src/metabase/visualizations/visualizations/CartesianChart/events.ts

```diff
@@ -130,7 +131,7 @@ export const getEventColumnsData = (
 
       return {
         key: col.display_name, // TODO: use the title from the viz settings
-        value,
+        value: value ?? t`(empty)`,
```

Comment:
we have `NULL_DISPLAY_VALUE` in `frontend/src/metabase/lib/constants.js`

Reply:
Fixed